### DISCUSSION TITLE
I am unable to sign the CLA. CLA assistant says there is no CLA to sign for forem/forem.

### DISCUSSION THREAD

**Discussion by modeandclay** [forem/forem#20655] [Q&A] [answered]
I am trying to open a PR for issue #20653.

I have read the contribution guide and followed the link https://cla-assistant.io/forem/forem

However, the page says that there is no CLA to sign and it could not find linked item for repo or owner. Any idea why? Is there any other way I can sign the CLA?

<img width="576" alt="Screenshot 2024-02-19 at 20 01 36" src="https://github.com/forem/forem/assets/46304222/6410d8e9-bf11-49d7-af52-219d1f5031a1">

**Comment by maestromac** [comment, marked as answer]
@codeandclay You only need to leave the comment like the following on the PR itself. 

example https://github.com/forem/forem/pull/20601#issuecomment-1933052977
